src/Offer/ReadModel/JSONLD/OfferLDProjector.php

```diff
@@ -418,7 +418,9 @@ protected function applyImageRemoved(AbstractImageRemoved $imageRemoved): ?JsonD
         );
 
         // Unset the main image if it matches the removed image
-        if (isset($offerLd->image) && $offerLd->{'image'} === $imageUrl) {
+        // @see https://jira.uitdatabank.be/browse/III-2201
+        // some historic ImageRemoved Events have http instead of https in the eventStore
+        if (isset($offerLd->image) && stristr($offerLd->{'image'}, ':') === stristr($imageUrl, ':')) {
```

Comment:
I had to read the documentation of `stristr` 🙂

If I understand correctly the `$offerLd->{'image'}` and `$imageUrl` get converted to string URLs without the protocol?

Reply:
Maybe a better command is 
```
Unset the main image if it matches the removed image. The comparison is done after stripping the protocol because historical data can have http instead of https
```

Reply:
Indeed comment 🙈